pkg/chain/config.go

```diff
@@ -0,0 +1,19 @@
+package eth
```

Comment:
It is fixed on `master`:
https://github.com/keep-network/keep-ecdsa/blob/d6bedeaac806bb03a8af050063570a9631c5a444/pkg/chain/chain.go#L4

Historically, `eth` was the only host chain and we had `btc` chain implementation for some functions that are no longer the responsibility of the client.